### PR TITLE
appendAllTrials for reportgen

### PR DESCRIPTION
**Pull request by dnoursi**

### CODE REVIEW

File: engine-comparison/report-gen/generate-report.go [hunk 1]
```diff
@@ -91,7 +91,21 @@ func handleFengine(fengine os.FileInfo, bmark_path string, desired_report_fname
 	return records
 }
 
-//func appendAllTrials(meta_records [][]string, records [][]string) [][]string {
+func appendAllTrials(meta_records [][]string, records [][]string) [][]string {
+
```
Comment (morehouse) [inline engine-comparison/report-gen/generate-report.go]: Remove unnecessary whitespace at beginning and end of functions.

File: engine-comparison/report-gen/generate-report.go [hunk 2]
```diff
@@ -91,7 +91,21 @@ func handleFengine(fengine os.FileInfo, bmark_path string, desired_report_fname
...
+	if len(meta_records) < len(records) {
+		meta_records = extendRecordsToTime(meta_records, len(records)-1, len(meta_records[0]))
+	}
+	// Store int to avoid recomputation
+	row_len := len(records[0])
+
+	for r, row := range records {
+		for c := 1; c < row_len; c++ {
+			meta_records[r] = append(meta_records[r], row[c])
```
Comment (morehouse) [inline engine-comparison/report-gen/generate-report.go]: I think you can remove this inner loop with something like `meta_records[r] = append(meta_records[r], row[1:]...)`.

Comment (dnoursi) [reply engine-comparison/report-gen/generate-report.go]: Woah! Cool, thanks, that code is exactly right

**Review by morehouse — APPROVED** [review]
